rpcd/playbooks/roles/rpc_setup_infrastructure/tasks/sshd.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
It would greatly simplify things if these tasks were in a small playbook rather then in it's own role.
Did you have something else in mind for the ``rpc_setup_infrastructure`` role? If not, I'd rather see these tasks in a small playbook. We could even add a variable in deploy.sh to toggle the MaxSessions adjustments. i.e. ``ADJUST_SSH_MAXSESSIONS``.